### ISSUE TITLE
Ares v114 in the history?

### ISSUE THREAD

**Issue by moon-chilled** [ares-emu/ares#7]
It would be cool if ares v114 were part of the commit history.  It would require a history rewrite, but trivial to rebase existing PRs, so not that big of a deal.  (Or, at least, being part of the same tree as an (orphan) branch; so all the source code is in a single, consolidated place.)

It can be downloaded @ the internet archive, [here](https://web.archive.org/web/20200625193611/https://ares.dev/releases)

**Comment by moon-chilled** [comment]
To do that history rewrite:

1. Make a new directory; git init, grab ares v114, git add, git commit -m ares114

2. `git pull --allow-unrelated-histories https://github.com/ares-emu/ares`

3. `rm -rf *`, overwrite all files with current versions from this repo.  (I do this with: `git clone https://github.com/ares-emu/ares ../aarreess; rsync -a ../aarreess/* .`)

4. `git add .`

5. `git commit`

This should make the first commit be ares v114, followed by all the commits in this repository, followed by a commit 'Merge https://github.com/ares-emu/ares into master'.  That last commit should be *empty*.  If all that works properly, then:

6. `git remote add origin https://github.com/ares-emu/ares-emu`

7. `git push -u origin master -f`

**Comment by Screwtapello** [comment]
ares v114 was originally left out because it was released under a restrictive Creative Commons licence rather than the ISC licence of v115.

Later, byuu gave me WIP snapshots and changelogs for all the work he'd done since higan v110, along with permission to distribute all of them under the ISC licence. I've stitched those together as a branch of the higan repository:

https://github.com/higan-emu/higan/tree/ares

My plan is to merge that branch back into the higan trunk, producing one codebase containing both the fixes and enhancements from ares (recording the development history) and the community improvements in higan. I also intend to land the currently-open PRs in this repository onto higan after the merge. However, as higan is under the GPLv3, the resulting combination will still be GPLv3.

I will probably leave this repository here as a source for people who want to use ares' code under the ISC licence alone. I should definitely update the README to make it clear to people what's going on — let me file an issue about that to remind myself.
